python/tvm/relax/transform/transform.py

```diff
@@ -303,6 +303,26 @@ def LambdaLift() -> tvm.ir.transform.Pass:
     return _ffi_api.LambdaLift()
 
 
+def LazyGetInput() -> tvm.ir.transform.Pass:
+    """A pass that requests inputs lazily
```

Comment:
I think it would be preferable to have more detail in the doc comment as to what exactly the output of the pass will look like (the same is true for `LazySetOutput` too).

Reply:
Good call, and I've updated the docstrings for each.